local_server/telemetry_server/server.py

```
@@ -0,0 +1,405 @@
+@app.post("/api/v1/telemetry", response_model=SubmitResponse)
+async def submit_telemetry(request: Request):
+    """接收遥测数据。验证流程：body 大小 → 时间戳 → HMAC 签名 → 速率限制 → 存储。"""
+    # Body 大小
+    body_bytes = await request.body()
+    if len(body_bytes) > MAX_BODY_SIZE:
+        raise HTTPException(413, "Payload too large")
+
+    try:
+        body_json = body_bytes.decode("utf-8")
+        submission = model_from_json(TelemetrySubmission, body_json)
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)